root/user/edits.tt

```diff
@@ -27,6 +27,6 @@
     END ~%]
 
 [% WRAPPER 'user/profile/layout.tt' title=page_title %]
-   <h2>[% page_heading %]</h2>
+   <h2>[% page_heading | html %]</h2>
```

Comment:
Because the above `param` is used both for the title and the heading, I had to add the encoding for the heading.